dotcom-rendering/src/palette.ts

```diff
@@ -48,6 +48,12 @@ const starRatingBackgroundColourLight = (): string =>
 const starRatingBackgroundColourDark = (): string =>
 	sourcePalette.brandAlt[200];
 
+const tableOfContentsLight = (): string => sourcePalette.neutral[7];
+const tableOfContentsDark = (): string => sourcePalette.neutral[86];
```

Comment:
Is this true for every article background? I believe the table of contents are most likely to be used on Explainer articles, which have a pinkish background.

Reply:
I believe so! these are the colours that a currently in use in DCR and AR. Also, if I remember right,  table of contents no longer appears on explainer or analysis articles since this change https://github.com/guardian/dotcom-rendering/pull/8607/files